.gitignore

```diff
@@ -4,7 +4,8 @@
 *.graphml
 process/data/input/*.csv
 validation/edge/data
-validation/poi/data
+validation/destination/data
```

Comment:
will we still need this line in the ignore file, given that the data have been moved to `/validation/data/` now?

Reply:
No, I am keeping it while we transition though.